ray-operator/config/samples/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-sample
+spec:
+  # submissionMode specifies how RayJob submits the Ray job to the RayCluster.
+  # The default value is "K8sJobMode", meaning RayJob will submit the Ray job via a submitter Kubernetes Job.
+  # The alternative value is "HTTPMode", indicating that KubeRay will submit the Ray job by sending an HTTP request to the RayCluster.
+  # The current value is "InteractiveMode", meaning that it will wait for user to submit job and provide the job submission ID
+
+  # shutdownAfterJobFinishes specifies whether the RayCluster should be deleted after the RayJob finishes. Default is false.
+  # shutdownAfterJobFinishes: false
+
+  # ttlSecondsAfterFinished specifies the number of seconds after which the RayCluster will be deleted after the RayJob finishes.
+  # ttlSecondsAfterFinished: 10
+
+  # activeDeadlineSeconds is the duration in seconds that the RayJob may be active before
+  # KubeRay actively tries to terminate the RayJob; value must be positive integer.
+  # activeDeadlineSeconds: 120
```

Comment:
Most of the comments in this file can be removed. We don't need to repeat them across multiple files. Only those not included in `ray-operator/config/samples/ray-job.sample.yaml` are worth keeping.

Reply:
I cleaned up the comments except the comment for `InteractiveMode`